Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM node:16.15-buster-slim
+FROM node:16.14.2-buster-slim
```

Comment:
This seems to have fixed it 🤷 . Here are the packaged versions:

```sh
$ node --version
v16.14.2
$ npm --version
8.5.0
```

Previously it was using:

```sh
$ node --version
v16.15.1
$ npm --version
8.11.0
```

Reply:
Seems like it could be either of these issues. Both are `8.6.0`+ problems as well which lines up with the `npm` version available in each base image.

 - https://github.com/npm/cli/issues/4769
 - https://github.com/npm/cli/issues/4996